src/annotator/integrations/pdf.tsx

```diff
@@ -259,6 +260,24 @@ export class PDFIntegration
     this._features.on('flagsChanged', () => {
       this.emit('supportedToolsChanged', this.supportedTools());
     });
+
+    // Override default behavior of links inside the PDF.
+    this._pdfViewer.viewer.addEventListener('click', event => {
+      // Make links in the PDF open in a new tab. This avoids accidentally
+      // navigating away from the PDF when trying to perform annotation actions.
+      const target = event.target as HTMLElement;
+      if (target instanceof HTMLAnchorElement) {
+        target.target = 'blank';
+      }
+
+      // Disable the link entirely if there is a highlight where the click
+      // happened. This avoids triggering the link when the user is trying to
+      // focus a highlight.
+      const highlights = getHighlightsFromPoint(event.clientX, event.clientY);
+      if (highlights.length > 0) {
+        event.preventDefault();
```

Comment:
I realize we're not actually checking whether the element is a link here, so this change actually prevents default behavior of any other element types if the user somehow manages to highlight them. I think that's probably the right thing to do. It could affect form controls in a PDF for example, although that's something that doesn't often come up.